qldbdriver/result.go

```diff
@@ -21,15 +21,17 @@ import (
 
 // Result is a cursor over a result set from a QLDB statement.
 type Result struct {
-	ctx          context.Context
-	communicator qldbService
-	txnID        *string
-	pageValues   []*qldbsession.ValueHolder
-	pageToken    *string
-	index        int
-	logger       *qldbLogger
-	ionBinary    []byte
-	err          error
+	ctx               context.Context
+	communicator      qldbService
+	txnID             *string
+	pageValues        []*qldbsession.ValueHolder
+	pageToken         *string
+	index             int
+	logger            *qldbLogger
+	ionBinary         []byte
+	err               error
```

Comment:
Nit: I feel like `err` should be the last field.

Reply:
error is directly paired with ionBinary so I don't agree.

Reply:
Then we should put `ionBinary` as the 2nd last field.